saas/views/metrics.py

```diff
@@ -110,27 +110,17 @@ class PlansMetricsView(ProviderMixin, TemplateView):
     (as a count of subscribers per plan per month)
     """
 
-    template_name = 'saas/plan_metrics.html'
+    template_name = 'saas/revenue_metrics.html'
```

Comment:
I am not sure we can do that. The plan metrics page needs to display the list of plans (with the active status pin) on the page otherwise we cannot browse to edit them. Both  *saas/plan_metrics.html* and *saas/revenue_metrics.html* already derive from *saas/metrics_base.html*, which contains the common code.